packages/web/src/components/reposition/reposition-info/RepositionInfo.tsx

```diff
@@ -96,8 +95,7 @@ const RepositionInfo: React.FC<RepositionInfoProps> = ({
               placement="top"
               FloatingContent={
                 <ToolTipContentWrapper>
-                  The deposit ratio of the two tokens is determined based on the
-                  current price and the set price range.
+                  The estimated APR from swap fees is calculated based on the selected price range of the position.
```

Comment:
What about managing it as a constants variable?

Reply:
I'm going to refactor it after PR